openghg/retrieve/_access.py

```diff
@@ -773,10 +766,7 @@ def _scale_convert(data: Dataset, species: str, to_scale: str) -> Dataset:
 
     converter = scale_converter.loc[row]
 
-    if to_scale == converter["scale1"]:
-        direction = "2to1"
-    else:
-        direction = "1to2"
+    direction = "2to1" if to_scale == converter["scale1"] else "1to2"
```

Comment:
Not a big deal but I actually think the original version of this is more clear - I know its more lines but sometimes it's ok to be a bit more verbose for the sake of readability.

Reply:
I think ruff suggested this.. I was experimenting with it. It seems like it would be an upgrade to black and flake8. I'll keep note of the flags it uses for this kind of change. Anyway, happy to revert it.

Reply:
Ah ok makes sense - I'd prefer the former but may just be personal preference. Happy to leave this up to you.

Reply:
Since the condition isn't too long, I think I prefer the one line version... it makes it clear that the main point is to define `direction`. Sometimes the one line version is unreadable though, so I'll avoid automatically changing these.